### PR TITLE
#199 fix environment variable substitution issue

### PR DESCRIPTION
**Pull request by csun-cpointe**
We use curly braces for some of the ENV substitutions in our Dockerfiles, but because fabric8 does Maven property expansion, and by default looks for ${} to do that substitution, we were getting injection of host machine values instead of properly using the VM’s env. This PR is to change the environment variable substitution from ${X} to $X

### CODE REVIEW

File: extensions/extensions-docker/aissemble-hive/aissemble-hive-service/pom.xml [hunk 1]
```diff
@@ -32,59 +32,23 @@
...
+                        <image>
+                            <build>
+                                <args>
+                                    <METASTORE_VERSION>${version.hive.metastore}</METASTORE_VERSION>
+                                    <JARS_DIR>${dockerbuild.jars.directory}</JARS_DIR>
+                                </args>
+                            </build>
+                        </image>
+                    </images>
+                </configuration>
```
Comment (csun-cpointe) [inline extensions/extensions-docker/aissemble-hive/aissemble-hive-service/pom.xml:51]: I: clean up the duplicated execution configuration.

File: extensions/extensions-docker/aissemble-spark-infrastructure/pom.xml [hunk 2]
```diff
@@ -52,57 +52,23 @@
...
+                    <images>
+                        <image>
+                            <build>
+                                <args>
+                                    <JARS_DIR>${dockerbuild.jars.directory}</JARS_DIR>
+                                </args>
+                            </build>
+                        </image>
+                    </images>
+                </configuration>
```
Comment (csun-cpointe) [inline extensions/extensions-docker/aissemble-spark-infrastructure/pom.xml:71]: I: clean up the duplicated execution configuration by moving the configuration out of plugin execution

**Review by cpointe-ibllanos — APPROVED** [review]
Did a thorough search on the project and cannot find any additional places the curly braces need to be removed. LGTM.

**Review by Cho-William — APPROVED** [review]

File: extensions/extensions-docker/aissemble-spark-infrastructure/src/main/resources/docker/Dockerfile [hunk 3]
```diff
@@ -6,9 +6,9 @@ ARG DELTA_HIVE_CONNECTOR_VERSION
 LABEL org.opencontainers.image.source="https://github.com/boozallen/aissemble"
 
 RUN curl -L https://github.com/delta-io/connectors/releases/download/v${DELTA_HIVE_CONNECTOR_VERSION}/delta-hive-assembly_2.12-${DELTA_HIVE_CONNECTOR_VERSION}.jar \
-          -o "${SPARK_HOME}"/jars/delta-hive-assembly_2.12-${DELTA_HIVE_CONNECTOR_VERSION}.jar
+          -o "$SPARK_HOME"/jars/delta-hive-assembly_2.12-${DELTA_HIVE_CONNECTOR_VERSION}.jar
 ARG JARS_DIR
-ADD ${JARS_DIR}/* ${SPARK_HOME}/jars/
+ADD ${JARS_DIR}/* $SPARK_HOME/jars/
```
Comment (carter-cundiff) [inline extensions/extensions-docker/aissemble-spark-infrastructure/src/main/resources/docker/Dockerfile:11]: What about `${JARS_DIR}`?

Comment (csun-cpointe) [reply extensions/extensions-docker/aissemble-spark-infrastructure/src/main/resources/docker/Dockerfile:11]: The build argument can be replaced with ${}

Comment (carter-cundiff) [reply extensions/extensions-docker/aissemble-spark-infrastructure/src/main/resources/docker/Dockerfile:11]: How is this any different then the `$JARS_DIR` in `extensions/extensions-docker/aissemble-hive/aissemble-hive-service/src/main/resources/docker/Dockerfile`?

**Pull request merged**